core/modules/views/handlers/views_handler_field_bulk_form.inc

```diff
@@ -232,6 +237,9 @@ class views_handler_field_bulk_form extends views_handler_field {
       $options = $relationships[$this->options['relationship']];
       $data = views_fetch_data($options['table']);
       $base_table = $data[$options['field']]['relationship']['base'];
+    } else {
```

Comment:
This "else" should go to a new line ... (deja-vue? :stuck_out_tongue_winking_eye:)